packages/icons/src/utils.ts

```diff
@@ -0,0 +1,47 @@
+import fs from "node:fs/promises";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import { optimize } from "svgo";
+
+/**
+ * Optimizes the given SVG content and removes all fills so the color can be set via CSS.
+ */
+export function optimizeSvg(svgContent: string, path: string) {
+  const { data } = optimize(svgContent, {
```

Comment:
In the PR description, you wrote that you also change 25x24px files to 24x24 -> where does this happen?

Reply:
I had to do this manually so this is a one-time fix where we don't have code for :)

Reply:
ok, I misunderstood because it was a sub-point of "optimized with svgo" ^^
it's fine, ppl probably won't look up old commit messages anyways :)